__fixtures__/invalid-workspace-1/packages/package-two/src/index.ts

```diff
@@ -0,0 +1 @@
+console.log('Package two');
```

Comment:
Do we really need these source files? Technically we don't even look at them right?